.github/workflows/publish.yml

```diff
@@ -33,7 +33,6 @@ jobs:
         with: 
           args: publish
         env:
-          ORG_GRADLE_PROJECT_publishUser: ${{ secrets.BINTRAY_USER }}
-          ORG_GRADLE_PROJECT_publishApiKey: ${{ secrets.BINTRAY_API_KEY }}
-
-
+          ORG_GRADLE_PROJECT_artifactory_contextUrl: ${{ secrets.ARTIFACTORY_CONTEXT_URL }}
+          ORG_GRADLE_PROJECT_artifactory_user: ${{ secrets.ARTIFACTORY_PUBLISH_USER }}
+          ORG_GRADLE_PROJECT_artifactory_password: ${{ secrets.ARTIFACTORY_PUBLISH_TOKEN }}
```

Comment:
Did we add these secrets in GitHub secrets?

Reply:
We didn't!

Reply:
not yet. I don't have admin rights. will ask one of the github admins to do so.

Reply:
these will be used only when we cut a new release.

Reply:
I'll add these this morning - I need to go generate a publish user and a token for them in artifactory too.

Reply:
![image](https://user-images.githubusercontent.com/45047841/113874133-50a12e00-976a-11eb-991f-4b2c36ddb2b1.png)
OK, should be available. Let's see if I set it up right...